backend/migration.js

```diff
@@ -0,0 +1,229 @@
+const path = require('path');
+const Umzug = require('umzug');
+const _ = require('lodash');
+
+function onMigrationEnd(exitCode) {
+    // eslint-disable-next-line no-process-exit
+    process.exit(exitCode);
+}
+
+function getCurrent(executed) {
+    return _.chain(executed).last().get('file').value() || '<NO_MIGRATIONS>';
+}
+
+/**
+ * Runs migration script
+ *
+ * @param {Object} loggerFactory initialized logger factory
```

Comment:
Maybe just for clarity add reference to the logger factory - `logger.js` file?

Reply:
Added